tests/indent/php/issue-4848.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+if (
+) {
+}
+
+return (
+);
+
+return true
+    ;
+
+return fn () => (
+);
+
+return fn (
+) => (
+);
+
+return function (
+) {
+};
+
+return function () {
+};
+
+return match (
+) {
+};
+
+return match () {
+        // double indent
```

Comment:
I don't think it's intended for this to have double indent :thinking:

Reply:
Good catch, I forgot to remove this---it was double indented before the fix